steps/appaloosa/1.0.0/step.yml

```diff
@@ -0,0 +1,167 @@
+      your store, and install your application. For all future runs of this step,
+      remove your email address and setup this step as a registered user.
+    title: User Email Address
+- APPALOOSA_API_KEY: null
+  opts:
+    description: '***Only if you are already registered on Appaloosa***'
+    is_dont_change_value: false
+    is_expand: true
+    is_required: false
+    summary: This is your unique Appaloosa API Key that can be found in your user's
+      profile on Appaloosa
+    title: User API Key
+- STORE_ID: null
+  opts:
+    description: |-
+      ***Only if you are already registered on Appaloosa***
+
+      This is the store id of the store you want your application uploaded to.
+    is_dont_change_value: false
+    is_expand: true
+    is_required: false
+    summary: You can find this value in your store settings
+    title: Store ID
+- SCREENSHOT1: null
+  opts:
+    description: URL (jpg or png)
+    is_dont_change_value: false
+    is_expand: true
+    is_required: false
+    summary: Enter the url of your first screenshot
+    title: SCREENSHOT 1
+- SCREENSHOT2: null
+  opts:
+    description: URL (jpg or png)
+    is_dont_change_value: false
+    is_expand: true
+    is_required: false
+    summary: Enter the url of your second screenshot
+    title: SCREENSHOT 2
+- SCREENSHOT3: null
+  opts:
+    description: URL (jpg or png)
+    is_dont_change_value: false
+    is_expand: true
+    is_required: false
+    summary: Enter the url of your third screenshot
+    title: SCREENSHOT 3
+- SCREENSHOT4: null
+  opts:
+    description: URL (jpg or png)
+    is_dont_change_value: false
+    is_expand: true
+    is_required: false
+    summary: Enter the url of your fourth screenshot
+    title: SCREENSHOT 4
+- SCREENSHOT5: null
+  opts:
+    description: URL (jpg or png)
+    is_dont_change_value: false
+    is_expand: true
+    is_required: false
+    summary: Enter the url of your fifth screenshot
+    title: SCREENSHOT 5
+- DESCRIPTION: null
+  opts:
+    description: Description or changelog for your application
+    is_dont_change_value: false
+    is_expand: true
+    is_required: false
+    summary: This is the description for your application. You can have a different
+      description for each version for your application.
+    title: Description
+- GROUP_IDS: null
+  opts:
+    description: |-
+      ***Only for registered users. ***
+
+      Groups ids of the groups you want your application published to. If you do not specify at least one group id, you will have to change the visibility directly on Appaloosa website.
+    is_dont_change_value: false
+    is_expand: true
+    is_required: false
+    summary: You can target specific groups on Appaloosa and publish your application
+      only to the users of these groups.
+    title: Group IDs
+outputs:
+- APPALOOSA_DEPLOY_IPA: ""
+  opts:
+    title: BINARY_PATH
+- APPALOOSA_API_KEY: ""
```

Comment:
why do you export the `APPALOOSA_API_KEY`? Checking your Step's code you explicitly **don't** want to expose these as outputs.

In general: `outputs` should contain only the Environment Variables which are **generated** by the step and **shared** with other steps. Checking your Step's code I think you don't have any, so you should remove the whole `outputs` section.